app/assets/stylesheets/application.tailwind.css

```diff
@@ -1,24 +1,2 @@
-@tailwind base;
```

Comment:
didn't work with these in, even if I switched to @import; doesn't seem to affect page so have removed for now